From f84ad84b4d0022ee93116130728688158972ca99 Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Thu, 3 Nov 2022 13:57:04 +0100
Subject: [PATCH] Don't force static linkage

---
 setup.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/setup.py b/setup.py
index 68c4539..a9c24f7 100644
--- a/setup.py
+++ b/setup.py
@@ -296,8 +296,6 @@ def awscrt_ext():
 
     else:  # unix
         # linker will prefer shared libraries over static if it can find both.
-        # force linker to choose static variant by using using "-l:libcrypto.a" syntax instead of just "-lcrypto".
-        libraries = [':lib{}.a'.format(x) for x in libraries]
         libraries += ['rt']
 
         # hide the symbols from libcrypto.a
-- 
2.37.1 (Apple Git-137.1)

