From 2088139deefaf6e0ba5d75eba7fe377db6e1ae0a Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Mon, 14 Jun 2021 14:13:08 +0200
Subject: [PATCH 2/2] Remove Werror from compile arguments

---
 setup.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index a7e7c09..1da038f 100644
--- a/setup.py
+++ b/setup.py
@@ -310,8 +310,7 @@ def awscrt_ext():
         extra_link_args += ['-pthread']
 
     if distutils.ccompiler.get_default_compiler() != 'msvc':
-        extra_compile_args += ['-Wextra', '-Werror', '-Wno-strict-aliasing', '-std=gnu99']
-        extra_link_args += ['-Wl,-fatal_warnings']
+        extra_compile_args += ['-Wextra', '-Wno-strict-aliasing', '-std=gnu99']
 
     return setuptools.Extension(
         '_awscrt',
-- 
2.32.1 (Apple Git-133)

